### PR TITLE
add gokv cli tool

### PR DESCRIPTION
**Pull request by abelharisov**
A prototype of CLI tool for getting/setting keys for issue https://github.com/philippgille/gokv/issues/6

### CODE REVIEW

**Comment by philippgille** [comment]
Hi @abelharisov,

Thank you already for the contribution! I just skimmed over the code and it looks very interesting. I'll have a more detailed look a little later, maybe I have some time on the upcoming weekend.

Could you please change the target branch from `master` to `develop`? That would be great.

Kind regards,
Philipp

File: gokv/main.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+func main() {
+	app := cli.NewApp()
+	app.Usage = "A cli tool for getting and setting key values"
+	app.UsageText = fmt.Sprint(app.Name, " get <KEY>\n   ", app.Name, " set <KEY> <VALUE>")
+	app.HideVersion = true
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
```
Comment (philippgille) [inline gokv/main.go]: I didn't use `urfave/cli` yet, but saw there's a configuration for `StringFlag` called `TakesFile` and related `FilePath` (see https://godoc.org/github.com/urfave/cli#StringFlag). Could that be useful here?

File: gokv/go.mod [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module github.com/philippgille/gokv/gokv
```
Comment (philippgille) [inline gokv/go.mod]: What do you think about moving the module to `github.com/philippgille/gokv/cmd/gokv`? And the `gen` module could live under the cmd path besides the gokv path.

That would especially make the tagging / releasing nicer. When tagging/releasing submodules, the current way the tag would probably have to be `gokv/v0.1.0`, which could be confused with the top level module that only contains the interface. `cmd/gokv/v0.1.0` makes the purpose of the module more clear.

**Comment by philippgille** [comment]
Hi @abelharisov, did you have time to look at my PR comments?

**Comment by abelharisov** [comment]
Hi! I didn't have a chance to look at PR comment yet. But I'll look at them for sure later.

**Comment by abelharisov** [comment]
Unfortunately, I can't find time to look and fix your PR comments, so I close my PR.

**Pull request closed without merging**